content/blog/azure-deployment-environments/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Pulumi + Azure Deployment Environments: Better Together for Enterprise Developers"
+allow_long_title: true
+date: 2024-05-21
+draft: false
+meta_desc: Author Azure Deployment Environments definitions with Pulumi using your favorite programming language.
+meta_image: meta.png
+authors:
+    - mikhail-shilkov
+tags:
+    - azure
+    - announcement
+    - platform-engineering
+---
+
+We are excited to announce the support for authoring [Azure Deployment Environments (ADE)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-environments) environment definitions in Pulumi Infrastructure as Code (IaC) empowering developers to self-serve app infrastructure required to deploy and test cloud-based applications. With Pulumi support, you can now manage your Azure resources in these environments using the same familiar programming model and the full power of our IaC platform.
+
```

Comment:
Consider adding the 
```html 
<!--more-->
``` 
here instead per the [blogging suggestion checklist item](https://github.com/pulumi/docs/blob/master/BLOGGING.md#publishing-check-list)